src/components/SearchBox.js

```diff
@@ -53,6 +54,9 @@ class SearchBox extends Component {
             }}
             onKeyUp={e => this.handleEnter(e)}
             title={intl.formatMessage({ id: "Search in name and description" })}
+            onBlur={e => {
```

Comment:
With the changes in this file searchbox.js you will encounter two problems:
- It will break existing search functionality on rasters
- It will get a merge conflict with master
Maybe it would be better to revert this file to original.
Then merge master in "hoan-scenario_basic-setup"
Then merge "hoan-scenario_basic-setup" in this branch.
Then see if the searchbar.js component fullfills your needs.
If not you can change it, but also keep testing raster page.

Reply:
Ah these are not changes made by me. I had the previous master branch and copied the searchbar.js from madeleine's branch to use it here. I probably would only need to delete this file and merge master back into this branch.

Reply:
Ah you are right. Your work flow is more correct. I should merge master into the base branch first and then merge the base branch into this branch.